app/org/sagebionetworks/bridge/StudyConsentBootstrapper.java

```diff
@@ -36,13 +37,17 @@ public StudyConsentBootstrapper(StudyService studyService, StudyConsentService s
             Study study = new DynamoStudy();
             study.setName("Test Study");
             study.setIdentifier("api");
+            study.setSponsorName("Sage Bionetworks");
             study.setMinAgeOfConsent(18);
             study.setResearcherRole("api_researcher");
             study.setConsentNotificationEmail("bridge-testing+consent@sagebridge.org");
+            study.setTechnicalEmail("bridge-testing+technical@sagebridge.org");
+            study.setSupportEmail("support@sagebridge.org");
```

Comment:
Also, bridge-testing@sagebase.org, not sagebridge.org. (Unless Xa set that up too.) Otherwise, we'll get a lot of bounces from SES.